docs/config/header.html

```diff
@@ -27,7 +27,26 @@
   <script type="text/javascript" src="$relpath^dynsections.js"></script>
   $treeview
   $search
-  $mathjax
+  <script type="text/x-mathjax-config">
+      window.MathJax = {
+      tex: {
+      tags: "ams",
+      inlineMath: [['^^','^^'], ['\\(', '\\)']],
```

Comment:
do we use `^^` anywhere? Doxygen seems to generate `\(`, and it's also the MathJax default, so we may not need this line at all.

Reply:
No, I'm removing it